packages/node/src/indexer/fetch.service.ts

```diff
@@ -213,6 +215,50 @@ export class FetchService implements OnApplicationShutdown {
     await this.getBestBlockHead();
   }
 
+  @Interval(BLOCK_TIME_VARIANCE * 10000)
```

Comment:
The interval isn't related to block time logically. If you think 50s is a reasonable value, just make a const and assign 50,000 to it.